### PR TITLE
chore(*): update import pkg path & readme

### PR DESCRIPTION
**Pull request by baiyutang**
- update `opensergo-go/pkg/proto/` path
- update shell env variable
- upgrade dependent package version

### CODE REVIEW

**Comment by baiyutang** [comment]
@li-jin-gou

**Review by li-jin-gou — APPROVED** [review]

**Comment by li-jin-gou** [comment]
CI need to solve

**Review by li-jin-gou — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -4,9 +4,9 @@ go 1.16
 
 require (
 	github.com/alibaba/sentinel-golang v1.0.4
-	github.com/cloudwego/kitex v0.3.4
+	github.com/cloudwego/kitex v0.4.3
 	github.com/cloudwego/kitex-examples v0.1.0
-	github.com/opensergo/opensergo-go v0.0.0-20220331070310-e5b01fee4d1c
+	github.com/opensergo/opensergo-go v0.0.0-20221129091737-554d5c0b9105
```
Comment (GuangmingLuo) [inline go.mod:9]: 名字好像换成 https://github.com/opensergo/opensergo-go-sdk 这个了

Comment (baiyutang) [reply go.mod:9]: image
好像不用改，库名变了，但是 包名没变

**Review by GuangmingLuo — APPROVED** [review]

**Review by GuangmingLuo — APPROVED** [review]
/LGTM

**Pull request merged**